### ISSUE TITLE
v1.23.2 on DX12 not showing up?

### ISSUE THREAD

**Issue by FeralRage7** [Lailloken/Lailloken-UI#51]
A few times the game started the script asked to rescan for fresh install but now I've restarted the game and the script and the script's archnemesis helper just does not show up at all. I can type /llk to bring up the menu and other functions (notepad) seems to be functional. Delete completely and maybe redownload? I am on Windowed Fullscreen, 1920x1080.

**Comment by Lailloken** [comment]
You can redownload it into a separate folder and try it from there. If that solves it, try editing the ini\config - archnemesis.ini file and removing the lines with "color1=" and "color2=" in your original script folder. Save the file and relaunch the script.

**Comment by FeralRage7** [comment]
- I've used the script for most of archnemesis from when I found it, must have been over a month ago
- Sorry, the script in the corner works and I bring up the menu, the timer and the notepad and before I hid it, the bottom left UI element was there.
- redownloaded and scanned and all is working. I lost my favorites list (the recipes I am searching for) but I'll redo that too. 

Feel free to close and thanks again!
